starter-code/index.html

```diff
@@ -17,16 +17,16 @@
       specify the type!
     - Insert correct placeholder properties based on our Model.
     -->
-    <script id=" " type=" ">
-      <article class="clearfix" data-author=" " data-category=" ">
+    <script id="articleTemplate" type="text/x-handlebars-template">
+      <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
         <header>
-          <h1> </h1>
+          <h1>{{title}}</h1>
           <div class="byline">
-            By <address><a href=" "> </a></address>
+            By <address><a href="{{authorUrl}}">{{author}}</a></address>
             {{publishStatus}}
```

Comment:
Nice job referencing the properties on the article instance passed to the function that was returned upon compiling your template (that's a mouthful.. haha)!